docs/src/modules/components/AppNavDrawer.js

```diff
@@ -207,19 +206,18 @@ function AppNavDrawer(props) {
         </SwipeableDrawer>
       ) : null}
       {disablePermanent || mobile ? null : (
-        <Hidden lgDown implementation="css">
-          <Drawer
-            classes={{
-              paper: classes.paper,
-            }}
-            variant="permanent"
-            open
-          >
-            <PersistScroll slot="side" enabled>
-              {drawer}
-            </PersistScroll>
-          </Drawer>
-        </Hidden>
+        <Drawer
+          classes={{
+            paper: classes.paper,
+          }}
+          variant="permanent"
+          sx={{ display: { xs: 'none', lg: 'block' } }}
```

Comment:
question: Assuming we just want to migrate from `<Hidden lgDown />`. Do we need `lg: 'block'` here? To me it isn't clear what the display is above `lg` or why we even need to differentiate between above and below `lg`. 

Documentation for this is lacking so whatever is the takeway, I'll incorporate into the docs.

Reply:
So I eventually found out how these works from the docs and it seems to me that `{ xs: 'none', lg: 'block' }` is not equivalent to `lgDown` if the screen width is below `xs`.
Wouldn't `{ display: 'none', lg: { display: 'none' }}` actually be equivalent? 
Though maybe `lg` isn't recognized and we would need `{ [theme.breakpoints.down('lg')]: { display: 'none' }`.

Reply:
Interesting feedback, `xs` is meant to be `width: 0px` but it's not very clear. I wonder how we could fix this. I have seen some name it differently, like `base`.

Reply:
+1 for 
```
{ [theme.breakpoints.down('lg')]: { display: 'none' } }
```

it's much more clear. Otherwise, I personally have hard time to follow what will be the end result.

Reply:
@mnajdova Then I guess it's a matter of habit and we should seriously consider #25846.

I'm biased, I'm used to thinking "mobile-first", `sx={{ display: { xs: 'none', lg: 'block' } }` reads naturally to me.

Reply:
Maybe, my point was more, if I see `lgDown`, I would be able to immediately replace it with:
```
{ [theme.breakpoints.down('lg')]: { display: 'none' } }
```
without thinking about it... 🤷 Otherwise, it would require some mental exercise 😄

Reply:
Why not add support for props like `xxDown` and `xxUp` in the `sx` prop? It would translate nicely with the props of `Hidden`.

> I'm biased, I'm used to thinking "mobile-first", hence why sx={{ display: { xs: 'none', lg: 'block' } } reads naturally to me.

For me it's not natural. We have a dilemma between users that think mobile-first and desktop-first.

> { [theme.breakpoints.down('lg')]: { display: 'none' } }

I changing to this as I think it's more readable. However, I can't use it outside a component, like [here](https://github.com/mui-org/material-ui/pull/26135/files#diff-2546e6fcedcd17ccc8a309f7c9d667fab518968c20f0f9e8cb9498275543a9ccR311).

Reply:
Anyway, not a blocker, I think it's personal preference, I wouldn't change the code to be honest. @eps1lon can we resolve?